help/destinations/catalog/email-marketing/salesforce-pardot.md

```diff
@@ -0,0 +1,203 @@
+---
+keywords: email;Email;e-mail;email destinations;salesforce;salesforce pardot;salesforce marketing cloud account engagement;salesforce pardot destination
+title: Salesforce Pardot
+description: The Salesforce Pardot (now renamed as Marketing Cloud Account Engagement) destination allows you to export your account data and activate it within Salesforce Pardot for your business needs.
+---
+# [!DNL Salesforce Pardot] connection
+
+[[!DNL Salesforce Pardot]](https://www.salesforce.com/products/marketing-cloud/overview/) *(now renamed as [!DNL Marketing Cloud Account Engagement])* allows you to capture and track leads by building templates, drag-and-drop builders; its powerful visual editor allows you to easily create custom landing pages, forms, and offers that attract potential leads for your sales teams. When someone engages with your forms, the lead is auto captured, and you get notified. Leads are also automatically scored and graded, so that those most likely to convert can be prioritized. Your forms can also trigger changes in lead scores upon conversion. [!DNL Salesforce Pardot] enables you to design lead tracks for all stages of the pipeline for targeted market segments and customer groups through email drip campaigns, lead management with nurturing and scoring and campaign segmentation.
+
```

Comment:
Suggestion for the overview -- I remember in the demo you briefly described how this destination is generally aimed at B2B use cases while other Salesforce destinations in our catalog are more aimed at B2C use cases. 

I think that information would be valuable here. Would you mind adding a blurb on that here?

Reply:
Added, also included a reference to the other connectors, if not required post back.